### PR TITLE
Welcome page update-- still need to dismiss on close and show only once.

### PR DESCRIPTION
**Pull request by bdsinger**
Announcing Welcome Page 0.9b-- without the ability to close! Yes, the X on the upper right does absolutely nothing!

[Update: the no-close and re-appear bugs have been fixed]

You can dismiss it by clicking on the background or typing something however. But wait, there's more...

It will reappear if your searchbox is empty! That's right, if you backspace after typing AIX, AIY or whatever, when the searchbox is back to being blank again, the Welcome Page is back again as well! Did you miss it?

Also, just noticed I reference "icon-welcome" in the final diff screen below, but never actually made/defined one of those.

WelcomePage

### CODE REVIEW

File: src/client/js/controller/welcome.js [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+
+      d.resolve();
+      return d;
+    })()
+      .then(() => {
+        // Update and display welcome.
+        this.view.help.setWelcomeContent(
+          'FunCoNN (Beta)',
+          [
+            '<b>Welcome to FuNCoNN (Beta): Functional Connectivity on NemaNode</b>',
```
Comment (lparsons) [inline src/client/js/controller/welcome.js:66]:
```suggestion
            '<b>Welcome to FunCoNN (Beta): Functional Connectivity on NemaNode</b>',
```

**Comment by bdsinger** [comment]
seems to work. If you miss it upon launch or reload, or dismiss it via close, it's gone!

**Review by lparsons — APPROVED** [review]
It looks good, though we need to change the the name on the page from `FuNCoNN` to `FunCoNN`. I made that as a suggestion if that's easier. Once that is done, I say we merge it and deploy.

**Review by hepcat72 — APPROVED** [review]
GEFM

**Pull request merged**